src/core/Application.js

```diff
@@ -54,16 +54,21 @@ export default class Application
          */
         this.ticker = new Ticker();
 
-        this.ticker.add(() =>
-        {
-            this.renderer.render(this.stage);
-        });
+        this.ticker.add(this.render, this);
 
         // Start the rendering
         this.start();
     }
 
     /**
+     * Render the current stage.
+     */
+    render()
```

Comment:
Maybe you want to call `app.stop()` to stop self-updating and then manually render `app.render()`. Could be useful in low-performance environment or where another ticker is driving the updates.